release/models/openconfig-extensions.yang

```diff
@@ -164,6 +170,23 @@ module openconfig-extensions {
       optimisations for sending or storing the corresponding data.";
   }
 
+  extension telemetry-atomic-exempt {
+    description
+      "The telemetry-atomic-exempt annotation is specified in the context
+      of a subtree (container, or list), and indicates that all nodes
```

Comment:
I think telemetry-atomic-exempt could also be defined for an individual node.  While I don't have a use case for this at the moment, it seems allowing it is useful.

Reply:
I agree. Updated the description to have "node".